plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -190,8 +192,13 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
         runReadAction {
             projectCoroutineScope(project).launch {
                 val result = downloadArtifact(job)
-                val summary = result.artifact?.summary ?: return@launch notifyUnableToShowSummary()
-                runInEdt { CodeModernizerSummaryEditorProvider.openEditor(project, summary) }
+                val summary = result.artifact?.summaryMarkdownFile ?: return@launch notifyUnableToShowSummary()
+                val summaryMarkdownVirtualFile = LocalFileSystem.getInstance().refreshAndFindFileByIoFile(summary)
+                if (summaryMarkdownVirtualFile != null) {
+                    runInEdt {
+                        FileEditorManager.getInstance(project).openFile(summaryMarkdownVirtualFile, true)
+                    }
+                }
```

Comment:
This is to open the `summary.md` as the raw MD file at end of job. We need the file links to be clickable and the only way to do that in the time we have left is to just open the raw MD and let IntelliJ handle it. This way all formatting done by Lin will just work automatically for both IDEs.